atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/Cells.java

```diff
@@ -111,7 +112,7 @@ public static Cell createLargestCellForRow(byte[] rowName) {
         return ret;
     }
 
-    public static <T> NavigableMap<byte[], SortedMap<byte[], T>> breakCellsUpByRow(Map<Cell, T> map) {
+    public static <T> NavigableMap<byte[], NavigableMap<byte[], T>> breakCellsUpByRow(Map<Cell, T> map) {
```

Comment:
This one is potentially dev-breaky (generics are invariant), though a quick look internally suggests that it has a few users outside AtlasDB, the last of which was updated in mid-2019 so I think it's fine. I'd suggest calling this one out in changelog to be certain.

Reply:
Added to the changelog.